src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -56,16 +56,17 @@ HRESULT OpenTerminalHere::Invoke(IShellItemArray* psiItemArray,
         siEx.StartupInfo.cb = sizeof(STARTUPINFOEX);
 
         // Append a "\." to the given path, so that this will work in "C:\"
-        auto cmdline{ wil::str_printf<std::wstring>(LR"-("%s" -d "%s\.")-", GetWtExePath().c_str(), pszName.get()) };
+        auto path{ wil::str_printf<std::wstring>(LR"-(%s\.)-", pszName.get()) };
+        auto cmdline{ wil::str_printf<std::wstring>(LR"-("%s" -d "%s")-", GetWtExePath().c_str(), path.c_str()) };

```

Comment:
We should consider migrating this old code to use the newer `fmt`. It would be:
```cpp
auto path = fmt::format(LR"-({}\.)-", pszName.get());
auto cmdline = fmt::format(LR"-("{}" -d "{}")-", GetWtExePath(), path);
```

Reply:
The man has just informed that `fmt` increases binary size by _a lot_ and `wil` was specifically chosen to drastically cut it down.